lib/tlog/es_json_reader.c

```diff
@@ -374,6 +374,10 @@ tlog_es_json_reader_refill_array(struct tlog_es_json_reader *es_json_reader)
     if (getenv("TLOG_CURL_USERPWD"))
         curl_easy_setopt(es_json_reader->curl, CURLOPT_USERPWD, getenv("TLOG_CURL_USERPWD"));
 
+    /* If $TLOG_CURL_VERBOSE is set enable verbose logging on the curl client. */
+    if (getenv("TLOG_CURL_VERBOSE"))
+        curl_easy_setopt(es_json_reader->curl, CURLOPT_VERBOSE, 1);
```

Comment:
This looks okay, but this might be better as an elastic search reader option that can be set in the configuration file, such as **--es-verbose** or **--es-curl-verbose**

Similar to:

~~~
   Elasticsearch reader options
       --es-baseurl=STRING
              Elasticsearch URL without query or fragment parts.

              STRING  is the base URL to request Elasticsearch through. Should not contain query (?...) or fragment
              (#...) parts.

       --es-query=STRING
              Elasticsearch query.

              STRING is the query string to send to Elasticsearch
~~~